mopidy/mpd/translator.py

```diff
@@ -90,17 +101,23 @@ def track_to_mpd_format(track, position=None, stream_title=None):
     return result
 
 
-def artists_to_mpd_format(artists):
+def concatenate_multiple_values(artists, attribute):
     """
-    Format track artists for output to MPD client.
+    Format track artist values for output to MPD client.
```

Comment:
MPD does in fact concatenate anything and everything. I've seen multiple values for track number, for example. Not sure that's ever useful, or indeed helpful, but yes the function is pretty generic so I'll change that.